src/rsz/test/repair_fanout7.ok

```diff
@@ -12,9 +12,9 @@ load DFF_X1 CK D 300000 metal3 1000
 [INFO ODB-0133]     Created 155 nets and 459 connections.
 worst slack -23.63
 [INFO RSZ-0094] Found 150 endpoints with setup violations.
-[INFO RSZ-0045] Inserted 78 buffers, 1 to split loads.
-[INFO RSZ-0041] Resized 527 instances.
-[INFO RSZ-0049] Cloned 7 instances.
+[INFO RSZ-0045] Inserted 41 buffers, 1 to split loads.
+[INFO RSZ-0041] Resized 312 instances.
+[INFO RSZ-0049] Cloned 4 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
 Repair timing output passed/skipped equivalence test
-worst slack -3.75
+worst slack -2.84
```

Comment:
If "The new code is not yet enabled" why are there changes to the results?

Reply:
General cleanup and restructuring of the code so we can actually switch over at some point.